pythonforandroid/recipes/libzmq/__init__.py

```diff
@@ -34,6 +34,7 @@ def build_arch(self, arch):
                 '--prefix={}'.format(prefix),
                 '--with-libsodium=no',
                 '--disable-libunwind',
+                '--disable-Werror',
```

Comment:
Hi @T-Dynamos !

Why `--disable-Werror` is needed? Do you have a trace or something else?

Reply:
Because of https://github.com/kivy/python-for-android/issues/2811#issue-1731366588 (also termux uses it)